apps/cms-v2/src/modules/providers/payloadcms/payloadcms.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Text } from "@saleor/macaw-ui/next";
+
+import { CMSProviderMeta } from "../cms-provider-meta";
+import logo from "./logo.png";
+
+export const PayloadCMS = {
+  formSideInfo: (
+    <Box>
+      <Text as="p" marginBottom={2}>
+        Configure the Payload CMS integration by providing required information.
+      </Text>
+      <Text as="p" marginBottom={2}>
+        Fields are not validated - ensure you enter correct values.
+      </Text>
+      <Text as="p" marginBottom={2}>
+        Consult docs for more information how to set up Payload CMS. TODO
```

Comment:
**Issue[^1][^2]**:
TODO left. 

[^1]: 🛑 Blocking the review. 
[^2]: Possible bug or not considered application state.